plf-packaging-resources/src/main/resources/RELEASE_NOTES.txt

```diff
@@ -136,18 +180,60 @@ However, if you still insist on skipping versions, we strongly advise to read al
 * Remove/Rename the file upgrade.properties
 * Restart the server. You're ready.
 
-== Upgrading from Platform 4.4.1 to ${org.exoplatform.platform.version} ==
-
-=== Description of the plugins used in this upgrade ===
+== Upgrading from Platform 4.4.4 to ${org.exoplatform.platform.version} ==
 
 === Major Components Upgrades ===
 
 * Tomcat 8.5
-Tomcat has been upgraded from 7.0 version to 8.5. The main changes related to this upgrade are:
-** conf/Catalina/localhost/context.xml.default has been removed, and all customizations related to Context have been gathered in conf/context.xml.
-** the property org.apache.catalina.startup.ContextConfig.jarsToSkip has been removed from conf/catalina.properties since it is not supported anymore and has been replaced by the JarScanning tag in conf/context.xml. The scanManifest has also been disabled (feature introduced in Tomcat 8.0) to avoid dependencies issues at startup.
-** war files are now exploded by default. This is a change introduced by Tomcat 8.0 (https://tomcat.apache.org/migration-8.html#Tomcat_8.0.x_noteable_changes). The variable SET EXO_TOMCAT_UNPACK_WARS can be set to false (defaults to true) in setenv-customize.sh|bat to not explode wars but this can lead to slower startup and performance issues, as mentioned in the link above.
-** Tomcat 8.5 comes with Servlet 3.1 (was 3.0 in Tomcat 7.0), JSP 2.3 (was 2.2 in Tomcat 7.0), EL 3.0 (was 2.2 in Tomcat 7.0)
+* JBoss EAP 7.0
+* Elasticsearch 5.6
+* Shindig 2.5
+* Infinispan 8.2
+* Upgrade to Groovy 2.4.12
+* Upgrade to JQuery 3.2.1
+* Upgrade to CKEditor to 4.7.2
+
+=== Description of the plugins used in this upgrade ===
+
+commons.upgrade.NodeTypeTemplateUpgradePlugin.enable=true
+
+#PLF-7186 Search Indexing Management page does not appear after PLF upgrade
+commons.upgrade.Upgrade-PortalData.enable=true
+
+# PLF-7253 Improve performances for the load space members.
+commons.upgrade.SpaceMemberAnyMembershipUpgradePlugin.enable=true
+
+#PLF-7169 : Upgrade from ISPN 5 to ISPN 8
+org.exoplatform.jcr.locks.force.remove=true
+
+=== Data migration ===
+
+The following data migration are automatically performed on the first startup of PLF ${org.exoplatform.platform.version}:
+* Settings and Notifications data are moved from JCR to JPA
+* Files are indexed in Elasticsearch instead of in the JCR
+* All space members are moved to JPA
```

Comment:
This will not be automatically updated only when we add upgrade.properties

Reply:
I removed the word "automatically". Is that ok ?